docs/nodle-wallets/nodle-cash/faq.mdx

```diff
@@ -66,9 +66,10 @@ Detailed instructions on [how to import your account from the app to the web](./
 
 ### How can I import my wallet to other wallets, like Polkadot JS, MetaMask, etc.?
 
-We do not support Polkadot.js and other wallets at this time.
+We only support the [Talisman](./nodle-wallets/talisman/introduction) wallet now.
```

Comment:
I may not know the context fully, but we shouldn't have any issues with Polkadot.js either.

Reply:
agreed, let's reword to something around the lines of "We recommend you use Talisman for managing your wallet on desktop. However, third parties such as Polkadot JS, SubWallet, or Nova should be compatible as well."

Reply:
@ETeissonniere But only if you have the SR key, right?

Reply:
ED compatibility will be restricted or require more steps. We have information on how to add an ED key to Polkadot JS but it requires quite a few more clicks

Reply:
Got it. I updated the text. @ETeissonniere can you please also check if internal links are corrent? I was not sure how to build them properly